pysat/tests/test_custom.py

```diff
@@ -198,10 +198,10 @@ def custom1(inst):
 
         self.testInst.custom.attach(custom1, 'add')
         self.testInst.load(2009, 1)
-        assert (self.testInst.data['doubleMLT'] == 2.0
-                * self.testInst['mlt']).all()
-        assert (self.testInst.data['tripleMLT'] == 3.0
-                * self.testInst['mlt']).all()
+        assert (self.testInst.data['doubleMLT'].values
+                == 2.0 * self.testInst['mlt'].values).all()
+        assert (self.testInst.data['tripleMLT'].values
+                == 3.0 * self.testInst['mlt'].values).all()
```

Comment:
The xarray object will store the new data under a different dimension, hence the error.